indico/modules/events/editing/client/js/components/FileTypeModal.jsx

```diff
@@ -56,6 +56,13 @@ export default function FileTypeModal({onClose, onSubmit, initialValues, header}
             )}
             <Form id="file-type-form" onSubmit={fprops.handleSubmit}>
               <FinalInput name="name" label={Translate.string('Name')} required />
+              <FinalInput
```

Comment:
if we go for `None` instead of `''`, we need `nullIfEmpty` here to convert empty strings to null